sql-cli/CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## 0.2.2
+
+### Bug fixes
+
+* Change validate to only check the connection of interest if the flag --connection is given #1370
+* Fix connection not found issue #1124
+* Improve error message when the remote connection can't be established #1366
+* Fix verbose logging #1368
+* Fix validate connection ID matching #1361
+* Support Airflow 2.2 #1162
+
```

Comment:
Are we not including: https://github.com/astronomer/astro-sdk/pull/1338 ?

Reply:
Thanks, @pankajkoti , I missed that - I just added the link to the ticket:
* Fix misplacement of generated DAGs by introducing the concept of global configuration #1230
Please, let me know your thoughts!